core/services/vrf/listener_v2.go

```diff
@@ -338,15 +348,16 @@ func (lsn *listenerV2) processRequestsPerSub(
 		// The ethcall will error if there is currently insufficient balance onchain.
 		maxLink, run, payload, gaslimit, err := lsn.getMaxLinkForFulfillment(maxGasPrice, req)
 		if err != nil {
+			rlog.Warnw("Unable to get max link for fulfillment, skipping request", "err", err)
```

Comment:
this warn looks redundant with the one inside getMaxLinkForFulfillment

Reply:
Hmm, yeah, it's redundant but has more log context (i.e more fields, tx hash, sub ID, request ID, etc.) Having all that context could be really useful, though we do pay for it with a bit of redundancy.